lib/roo_on_rails/checks/heroku/app_exists.rb

```diff
@@ -16,6 +16,10 @@ module Heroku
       # Output context:
       # - heroku.app.{env}: an app name.
       class AppExists < EnvSpecific
+        ACCEPTABLE_ENV_NAMES = {
+          'production' => ['prod', 'production'],
+          'staging'    => ['stg', 'staging']
```

Comment:
furthermore, do we want to ensure 'uniformity' - do we want to allow production to be named 'application-production' and staging to be named 'application-stag' or do we want to allow only valid sets:

- staging, production
- stag, prod
- test, prod